OpenMEEG/libs/OpenMEEGMaths/include/matrix.h

```diff
@@ -187,7 +187,7 @@ namespace OpenMEEG {
     }
     
     inline double Matrix::frobenius_norm() const {
-    #ifdef HAVE_LAPACK
+    #if defined(HAVE_LAPACK) && !defined(NO_DLANGE)
```

Comment:
lapacke_dlange is supported on new versions of MKL, but I cannot test it, because the machines on ci are only with 2013 version of MKL, so that was a fix to be able to use MKL not depending on the version.
And as I told you it is rarely used the frobenius norm (only in tests). I'll try to get a newer version of MKL to remove this workaround.